README.md

```diff
@@ -1,91 +1,52 @@
 
-# GEOS on GPU - Continuous Integration
+# Software Modernization Team: The Code Nebulae
 
-On-premise CI for the GPU ports of GEOS. Includes validation & benchmark worfklows.
+This is the team sandbox repository, storing early PoC, relevant benchmark data, staging `ci` workflows, etc.
+Shortname for the packge is `tcn`.
+
+🚧 This is a staging/PoC area for code, inherently every code here is unstable. 🚧
+
+Below is a quick summary of the tools/packages present in code. More information is package-level READMEs.
 
-## Validation status
+## `ci`
+
+On-premise CI for the GPU ports of GEOS. Includes validation & benchmark worfklows.
 
 Heartbeat insure the workflow to reach Discover is working.
 Validation capacities in the case of GEOS is stricly building & running the GPU-enabled version.
 Validation capacities for physics compares OACC and original Fortran on.
 
 | Validation                    | Status    |
 | ------------------------------------------ | --------- |
-| NCCS Discover Heartbeat                    | [![💓](https://github.com/GEOS-ESM/geosongpu-ci/actions/workflows/discover_heartbeat_nightly.yml/badge.svg)](https://github.com/GEOS-ESM/geosongpu-ci/actions/workflows/discover_heartbeat_nightly.yml) |
-| NCCS Discover GEOS Held-Suarez Validation  | [![HS](https://github.com/GEOS-ESM/geosongpu-ci/actions/workflows/discover_hs_nightly.yml/badge.svg)](https://github.com/GEOS-ESM/geosongpu-ci/actions/workflows/discover_hs_nightly.yml) |
+| NCCS Discover Heartbeat                    | [![💓](https://github.com/GEOS-ESM/smtn/actions/workflows/discover_heartbeat_nightly.yml/badge.svg)](https://github.com/GEOS-ESM/smtn/actions/workflows/discover_heartbeat_nightly.yml) |
```

Comment:
Note: It looks like GEOS-ESM/smtn does not exist...

Reply:
Good catch. That's what I get for renaming the rename...